### PR TITLE
Customize provided code

### PR DESCRIPTION
**Pull request by ei84136**
Edits the starter code as instructed for the portfolio project.

This is the first step of the creation of my portfolio project before the addition of extra pages.

### CODE REVIEW

File: portfolio/src/main/webapp/script.js [hunk 1]
```diff
@@ -17,7 +17,13 @@
  */
 function addRandomGreeting() {
   const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+      ['Run!',
```
Comment (kschon) [inline portfolio/src/main/webapp/script.js:20]: Style nitpick: When using a block-style array, put a break after the opening brace, like:

const my_arr = [
  'A',
  'B',
  'C',
];

For Google internal code, there would probably be a linter test that would require this.

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -17,7 +17,13 @@
...
+        'Don\'t blink.',
+        'I\'m planning on dyeing my hair pink.',
+        'I have a cat named Gracie.',
+        'It\'s dangerous to go alone! Take this.',
+        'You want weapons? We\'re in a library! Books! The best weapons in the world!',
```
Comment (kschon) [inline portfolio/src/main/webapp/script.js:25]: Dr. Who? There's a David Tenant episode in a library, and this sounds like it's from that.

File: portfolio/src/main/webapp/script.js (hunk 1 above)
Comment (ei84136) [reply portfolio/src/main/webapp/script.js:20]: Thank you for letting me know! I'll fix it and take a look at the style guides to keep an eye out for that in the future.

File: portfolio/src/main/webapp/script.js (hunk 2 above)
Comment (ei84136) [reply portfolio/src/main/webapp/script.js:25]: Right TV show!

**Pull request merged**